crates/compiler/builtins/bitcode/src/str.zig

```diff
@@ -1306,6 +1306,44 @@ test "countGraphemeClusters: emojis, ut8, and ascii characters" {
     try expectEqual(count, 10);
 }
 
+// Str.graphemes
+pub fn strGraphemes(string: RocStr) callconv(.C) RocList {
+    var list = RocList.allocate(@alignOf(u32), string.len(), @sizeOf(u32));
+    const graphemes = @ptrCast([*]RocStr, @alignCast(@alignOf(RocStr), list.bytes));
```

Comment:
For the purposes of this PR, all strings will be "small" strings which means they have a fixed size of 24 bytes (or 12 bytes on a 32-bit target) and you can get a pointer to the contents using `asU8ptr()`.
You probably want to create the RocStr using some helper function from str.zig where you pass the length in bytes. That will probably have some name like "allocate", I can't remember. For the size of strings you are dealing with, it will not allocate any heap memory and will just work.
Then you want to  get a pointer to the contents using `asU8ptr()` and write those into your RocList. Always increment through the RocList in increments of `sizeOf(RocStr)`.